.github/release-drafter.yml

```diff
@@ -50,7 +50,7 @@ exclude-labels:
 exclude-contributors:
   - 'dependabot'
   - 'dependabot[bot]'
-  - 'process-analytics-bot'
+  - ${{ vars.PA_BOT_EMAIL }}
```

Comment:
⚠️ AFAIK, the release-drafter configuration doesn't kown the GH Actions vars context
```suggestion
  - 'process-analytics-bot'
```